libtrellis/CMakeLists.txt

```diff
@@ -108,16 +108,23 @@ endif()
 
 find_package(Boost REQUIRED COMPONENTS program_options)
 
+get_property(LIB64 GLOBAL PROPERTY FIND_LIBRARY_USE_LIB64_PATHS)
+if ("${LIB64}" STREQUAL "TRUE")
+    set(LIBDIR "lib64")
+else()
+    set(LIBDIR "lib")
+endif()
+
 function(setup_rpath name)
   if(APPLE)
     set_target_properties(${name} PROPERTIES
                           BUILD_WITH_INSTALL_RPATH ON
-                          INSTALL_RPATH "@loader_path/../lib"
+                          INSTALL_RPATH "@loader_path/../${LIBDIR}/prjtrellis"
```

Comment:
I'd personally prefer to avoid the subfolder and any potential complications it might bring, unless it is a distro requirement.